lib/plugins/requestExpiry.js

```diff
@@ -4,26 +4,56 @@ var assert = require('assert-plus');
 var GatewayTimeoutError = require('restify-errors').GatewayTimeoutError;
 
 /**
- * A request expiry will use the headers to tell if the
- * incoming request has expired or not.  The header is
- * expected to be in absolute time since the epoch.
+ * A request expiry will use headers to tell if the
+ * incoming request has expired or not. There are two options for this plugin:
+ *  1. Absolute Time
+ *     * Time in Milliseconds since the Epoch when this request should be
+ *       considered expired
+ *  2. Timeout
```

Comment:
What about changing this slightly such that if the `starHeader` isn't passed in, the plugin starts counting from when the request was first received?